applications/dashboard/views/profile/edit.php

```diff
@@ -8,7 +8,7 @@
 <ul>
    <li class="User-Name">
       <?php
-         echo $this->Form->Label('Username', 'Name');
+         echo $this->Form->Label(T('Username'), 'Name');
```

Comment:
`Gdn_Form->Label()` already passes the argument through `T()` so this is redundant.